unittests/accents.py

```diff
@@ -0,0 +1,8 @@
+from plasTeX.TeX import TeX
+
+def test_accent():
+    input_data = r"""\"o"""
```

Comment:
```suggestion
    input_data = r'\"o'
```
That should be enough "